tools/lint-docstring

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+
+import argparse
+import importlib
+import sys
+from os import getcwd, listdir
+from os.path import isfile, join
+from typing import Dict, List, Tuple, cast
+
+
+developer_doc = join(getcwd(), 'docs', 'developer-file-overview.md')
+
+
+def get_docstrings() -> Tuple[Dict[str, str], Dict[str, List[str]]]:
+    root = join(getcwd(), 'zulipterminal')
+    folder_paths = {
+        'zulipterminal': root,
+        'zulipterminal/cli': join(root, 'cli'),
+        'zulipterminal/config': join(root, 'config'),
+        'zulipterminal/ui_tools': join(root, 'ui_tools')
+    }
+    total_files = {}
+    for folder, path in folder_paths.items():
+        total_files[folder] = [file for file in listdir(path)
+                               if(isfile(join(path, file))
+                               and file != '__init__.py')]
+    all_zt_files_dict = {}
+    for folder, files in total_files.items():
+        for file in files:
+            imported_file = importlib.import_module(
+                            f'{folder.replace("/",".")}.{file[:-3]}')
+            docstring = cast(str, imported_file.__doc__)
+            all_zt_files_dict[file] = docstring.strip().replace('\n', ' ')
+    return (all_zt_files_dict, total_files)
+
+
+def get_file_overview_doc_docstrings() -> Dict[str, str]:
+    with open(developer_doc, 'r') as file:
+        doc_data = file.readlines()
+
+    # table is present from line 6 to (length - 1)
+    computed_doc_dict = {}
+    for i in range(6, len(doc_data)):
```

Comment:
We may refactor the starting line index as well (so as to easily handle any changes in future). Something like
```python
TABLE_START_LINE = 6
```
would be great :)